### PR TITLE
Update spelling of “macOS”

### PR DESCRIPTION
**Pull request by sonicdoe**
If you’re okay with this change, I can update the other languages as well.

### CODE REVIEW

**Comment by JuanitoFatas** [comment]
> If you’re okay with this change, I can update the other languages as well.

This looks good, please include other languages 👍

**Comment by sonicdoe** [comment]
Done 👍

**Comment by olivierlacan** [comment]
@sonicdoe I apologize for causing this conflict but you might need to rebase against master since #1877 was merged. Thanks for your work. 🙇

File: ko/about/index.md [hunk 1]
```diff
@@ -136,7 +136,7 @@ end
 * 루비는 운영체제에 의존하지 않는 스레딩 기능을 제공합니다. 그리고 이는 루비가 동작하는 모든 플랫폼에 적용되어서 스레드를
   지원하지 않는 MS-DOS에서도 다중 스레드 프로그램을 작성할 수 있습니다.
 
-* 루비는 이식성이 매우 높습니다. 대부분 GNU Linux에서 개발되었지만 대부분의 UNIX, 맥 OS X, Windows, DOS, BeOS, OS/2 등에서 동작합니다.
+* 루비는 이식성이 매우 높습니다. 대부분 GNU Linux에서 개발되었지만 대부분의 UNIX, 맥 macOS, Windows, DOS, BeOS, OS/2 등에서 동작합니다.
```
Comment (yous) [inline ko/about/index.md]: '맥' is Mac. 😃 

```suggestion
* 루비는 이식성이 매우 높습니다. 대부분 GNU Linux에서 개발되었지만 대부분의 UNIX, macOS, Windows, DOS, BeOS, OS/2 등에서 동작합니다.
```

File: ko/documentation/installation/index.md [hunk 2]
```diff
@@ -175,7 +175,7 @@ $ pkg install runtime/ruby-18
 [ruby-build][ruby-build]는 다른 버전의 루비를 임의의 디렉터리에
 컴파일하고 설치할 수 있게 하는 [rbenv](#rbenv)의 플러그인입니다.
 ruby-build는 rbenv 없이 독자적으로 사용 할 수도 있습니다.
-맥 OS X, Linux나 다른 UNIX-계열 운영체제에서만 사용 가능합니다.
+맥 macOS, Linux나 다른 UNIX-계열 운영체제에서만 사용 가능합니다.
```
Comment (yous) [inline ko/documentation/installation/index.md]:
```suggestion
macOS, Linux나 다른 UNIX-계열 운영체제에서만 사용 가능합니다.
```

File: ko/documentation/installation/index.md [hunk 3]
```diff
@@ -184,7 +184,7 @@ ruby-build는 rbenv 없이 독자적으로 사용 할 수도 있습니다.
 [ruby-install][ruby-install]은 다른 버전의 루비를 임의의 디렉터리에
 컴파일하고 설치할 수 있게 합니다.
 루비 버전을 전환하는 [chruby](#chruby)와 형제 관계입니다.
-맥 OS X, Linux나 다른 UNIX-계열 운영체제에서만 사용 가능합니다.
+맥 macOS, Linux나 다른 UNIX-계열 운영체제에서만 사용 가능합니다.
```
Comment (yous) [inline ko/documentation/installation/index.md]:
```suggestion
macOS, Linux나 다른 UNIX-계열 운영체제에서만 사용 가능합니다.
```

File: ko/documentation/installation/index.md [hunk 4]
```diff
@@ -230,21 +230,21 @@ chruby는 [ruby-install](#ruby-install)로 설치된 루비뿐만
 [rbenv][rbenv]는 여러 종류의 루비를 설치할 수 있게 합니다.
 rbenv 자체는 루비 설치를 지원하지 않습니다만, [ruby-build](#ruby-build)라는
 유명한 플러그인에서 루비를 설치할 수 있습니다.
-rbenv, ruby-build 모두 맥 OS X, Linux나 다른 UNIX-계열 운영체제에서 사용가능합니다.
+rbenv, ruby-build 모두 맥 macOS, Linux나 다른 UNIX-계열 운영체제에서 사용가능합니다.
```
Comment (yous) [inline ko/documentation/installation/index.md]:
```suggestion
rbenv, ruby-build 모두 macOS, Linux나 다른 UNIX-계열 운영체제에서 사용가능합니다.
```

File: ko/documentation/installation/index.md [hunk 5]
```diff
@@ -230,21 +230,21 @@ chruby는 [ruby-install](#ruby-install)로 설치된 루비뿐만
...
 
 
 ### RVM ("Ruby Version Manager")
 {: #rvm}
 
 [RVM][rvm]은 시스템에 여러 종류의 루비를 설치하고 관리할 수 있게 해줍니다.
 RVM은 다른 gemsets도 관리해 줍니다.
-맥 OS X, Linux나 다른 UNIX-계열 운영체제에서만 사용 가능합니다.
+맥 macOS, Linux나 다른 UNIX-계열 운영체제에서만 사용 가능합니다.
```
Comment (yous) [inline ko/documentation/installation/index.md]:
```suggestion
macOS, Linux나 다른 UNIX-계열 운영체제에서만 사용 가능합니다.
```

File: ko/documentation/quickstart/index.md [hunk 6]
```diff
@@ -32,7 +32,7 @@ header: |
 
 irb(Interactive Ruby의 약자)를 시작하십시오.
 
-* **맥 OS X**를 사용한다면, 터미널 애플리케이션을 시작하여 `irb`라고
+* **맥 macOS**를 사용한다면, 터미널 애플리케이션을 시작하여 `irb`라고
```
Comment (yous) [inline ko/documentation/quickstart/index.md]:
```suggestion
* **macOS**를 사용한다면, 터미널 애플리케이션을 시작하여 `irb`라고
```

**Comment by sonicdoe** [comment]
@olivierlacan Thanks for the heads-up, it’s rebased now.
@yous Thanks for your help, I’ve incorporated your suggestions.

**Comment by JuanitoFatas** [comment]
@sonicdoe Fantastic work! Thanks Jakob and everyone who reviews, comments 🎉

**Pull request merged**